src/utils.ts

```diff
@@ -102,9 +102,20 @@ export const getFileIdRegexp = () => {
 
 export const FILE_ID_REGEXP = getFileIdRegexp();
 
-export const getFileTypesSetting = () => {
+export const getFileSearchExpression = (): string => {
```

Comment:
I'm not sure about the naming `getFileSearchExpression`. It could be changed to `getFindFilesGlob` to make it more in sync with VS Code API?

Reply:
`getFindFilesGlob` sounds better to me. Although we probably don't want to shadow the API in case this is what happens here (I don't fully get what you mean so might be talking nonsense).

Reply:
I agree, `findFilesGlob` sounds better. I renamed:

- `getFilesSearchExpression` to `getFindFilesGlob`
- `searchGlob` to `findFilesGlob`

This way it's consistent throughout the whole change.

> I don't fully get what you mean so might be talking nonsense

The second sentence in my comment only meant that VS Code API decided to name the option for finding files `getFindFilesGlob`. I think that it's good that we use the same naming if we don't have a particular reason to diverge.